Martin V\"ath <martin at mvath.de>:
Fix missing includes and function declarations of luvcview-0.2.6 which make
compilation fail on, for instance, gcc-14.
--- 1/v4l2uvc.c
+++ 1/v4l2uvc.c
@@ -25,6 +25,7 @@
 #include <math.h>
 #include <float.h>
 
+#include <SDL/SDL.h>
 #include <libv4l2.h>
 
 #include "v4l2uvc.h"
--- 1/v4l2uvc.h
+++ 1/v4l2uvc.h
@@ -90,13 +90,17 @@
 int uvcGrab(struct vdIn *vd);
 int close_v4l2(struct vdIn *vd);
 
+int check_videoIn(struct vdIn *vd, char *device);
+
 int v4l2GetControl(struct vdIn *vd, int control);
 int v4l2SetControl(struct vdIn *vd, int control, int value);
 int v4l2UpControl(struct vdIn *vd, int control);
 int v4l2DownControl(struct vdIn *vd, int control);
 int v4l2ToggleControl(struct vdIn *vd, int control);
 int v4l2ResetControl(struct vdIn *vd, int control);
+int v4l2ResetPan(struct vdIn *vd);
 int v4l2ResetPanTilt(struct vdIn *vd);
+int v4l2ResetTilt(struct vdIn *vd);
 int v4L2UpDownPan(struct vdIn *vd, short inc);
 int v4L2UpDownTilt(struct vdIn *vd,short inc);
 int v4L2UpDownPanTilt(struct vdIn *vd, short inc_p, short inc_t);
